sql/parser/eval.go

```diff
@@ -455,9 +455,12 @@ func EvalExpr(expr Expr, env Env) (Datum, error) {
 
 	case *CastExpr:
 		return evalCastExpr(t, env)
+
+	default:
+		return null, fmt.Errorf("eval: unsupported expression type: %T", expr)
 	}
 
-	return null, fmt.Errorf("unsupported expression type: %T", expr)
+	return null, fmt.Errorf("eval: unexpected expression: %T", expr)
```

Comment:
yeah the second return may as well be a panic to guard against a missing `return`?